eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -1,5 +1,27 @@
 <UsageData>
   <IgnorePatterns>
-    <UsagePattern IdentityGlob="*/*" />
+    <UsagePattern IdentityGlob="Microsoft.SourceBuild.Intermediate.*/*" />
+
+    <!-- These are used in-build for repo-tasks and should come from previously source built -->
+    <UsagePattern IdentityGlob="NuGet.Common/*6.2.2*" />
```

Comment:
Will we have to maintain these hardcoded versions?

Reply:
Why not use the versions from `Versions.props`?

Reply:
This isn't loaded by msbuild. They will need to be maintained, but I assume they will not update too often.

Reply:
Will we see an error if we update the related package versions but don't update these? Want to make sure we don't let it fall behind